### PR TITLE
support show full tables

### LINKED ISSUE #4697: Feature: Need support query: `show full tables`
**Summary**

Now not support show full tables. So some GUI can not be used.

For example: 

Query	/* ApplicationName=DBeaver 22.0.1 - Metadata */ SHOW FULL TABLES FROM db

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Support query `show full tables`

## Changelog

- New Feature
- Testing
- Documentation

## Related Issues

Fixes #4697

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by TCeason** [comment]
I will fixed the test err.

File: query/src/interpreters/interpreter_show_tables.rs [hunk 1]
```diff
@@ -40,18 +40,35 @@ impl ShowTablesInterpreter {
 
     fn build_query(&self) -> Result<String> {
         let database = self.ctx.get_current_database();
+        let showfull = self.plan.showfull;
         return match &self.plan.kind {
             PlanShowKind::All => {
-                Ok(format!("SELECT name FROM system.tables WHERE database = '{}' ORDER BY database, name", database))
+                if showfull {
```
[the remark this answers is not in the document]
Comment (TCeason) [reply query/src/interpreters/interpreter_show_tables.rs:46]: Ooooh, that's cool. I will modify.

File: docs/doc/30-reference/30-sql/40-show/show-tables.md [hunk 2]
```diff
@@ -34,46 +34,61 @@ mysql> SHOW TABLES;
...
 ```sql
-mysql> SHOW TABLES LIKE 'numbers_local';
-+---------------+
-| name          |
-+---------------+
-| numbers_local |
-+---------------+
+mysql> SHOW TABLES LIKE 'settings';
++------------+
+| table_name |
```
Comment (sundy-li) [inline docs/doc/30-reference/30-sql/40-show/show-tables.md:39]: Not `Tables_in_default` ?

File: docs/doc/30-reference/30-sql/40-show/show-tables.md [hunk 3]
```diff
@@ -7,15 +7,15 @@ Shows the list of tables in the currently selected database.
...
+SHOW [FULL] TABLES  [LIKE 'pattern' | WHERE expr | FROM 'pattern' | IN 'pattern']
 ```
 
 ## Examples
 
 ```sql
 mysql> SHOW TABLES;
 +---------------+
-| name          |
+| table_name    |
```
Comment (sundy-li) [inline docs/doc/30-reference/30-sql/40-show/show-tables.md:18]: https://www.db-fiddle.com/f/k2vZTW2y8Vm9AjUSTPkVGv/0

File: docs/doc/30-reference/30-sql/40-show/show-tables.md (hunk 2 above)
Comment (TCeason) [reply docs/doc/30-reference/30-sql/40-show/show-tables.md:39]: No, just `show full table` will modify the col name.

File: docs/doc/30-reference/30-sql/40-show/show-tables.md (hunk 3 above)
Comment (TCeason) [reply docs/doc/30-reference/30-sql/40-show/show-tables.md]: Fixed.

**Review by sundy-li — APPROVED** [review]

**Comment by TCeason** [comment]
@BohuTANG Please review~

**Pull request merged**